build.sbt

```diff
@@ -8,29 +8,38 @@ ThisBuild / organizationHomepage := Some(url("https://aiven.io/"))
 
 ThisBuild / resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
 
-val akkaVersion                 = "2.6.20"
-val akkaHttpVersion             = "10.2.10"
-val alpakkaKafkaVersion         = "3.0.1"
-val kafkaClientsVersion         = "3.4.0"
-val alpakkaVersion              = "4.0.0"
-val futilesVersion              = "2.0.2"
-val quillJdbcMonixVersion       = "3.7.2"
-val postgresqlJdbcVersion       = "42.6.0"
-val scalaLoggingVersion         = "3.9.5"
-val logbackClassicVersion       = "1.4.6"
-val declineVersion              = "2.4.1"
-val pureConfigVersion           = "0.17.2"
-val scalaTestVersion            = "3.2.15"
-val scalaTestScalaCheckArtifact = "scalacheck-1-17"
-val scalaTestScalaCheckVersion  = "3.2.15.0"
-val akkaStreamsJson             = "0.8.3"
-val diffxVersion                = "0.8.2"
-val testContainersVersion       = "0.40.14"
-val testContainersJavaVersion   = "1.17.6"
-val scalaCheckVersion           = "1.17.0"
-val scalaCheckOpsVersion        = "2.10.0"
-val enumeratumVersion           = "1.7.2"
```

Comment:
The reason behind this large diff block is due to scalafmt, i.e. since `scalaTestScalaCheckArtifact` was moved outside of the block the padding before `=` has changed. To verify this just change diff view in github UI to ignore whitespace.